modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java

```diff
@@ -168,14 +164,14 @@ private ReconciliationResult partitionReconciliationCheck(
     /** {@inheritDoc} */
     @Override public void parseArguments(CommandArgIterator argIter) {
         Set<String> cacheNames = null;
-        boolean fixMode = (boolean) FIX_MODE.defaultValue();
-        boolean verbose = (boolean) INCLUDE_SENSITIVE.defaultValue();
-        boolean console = (boolean) LOCAL_OUTPUT.defaultValue();
-        double loadFactor = (double) LOAD_FACTOR.defaultValue();
-        int batchSize = (int) BATCH_SIZE.defaultValue();
-        int recheckAttempts = (int) RECHECK_ATTEMPTS.defaultValue();
-        RepairAlgorithm repairAlg = (RepairAlgorithm) FIX_ALG.defaultValue();
-        int recheckDelay = (int) RECHECK_DELAY.defaultValue();
+        boolean fixMode = false;
```

Comment:
Why not to rename it to repair?
and use REPAIR.defaultValue() here?

Reply:
REPAIR.defaultValue() -> returns default strategy. If this argument exists, fixMode = true.